webpack.config.js

```diff
@@ -0,0 +1,57 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    library: 'AFRAME',
+    libraryTarget: 'umd',
+    path: path.resolve(__dirname, 'dist'),
+    publicPath: '/dist/',
+    filename: 'aframe-master.js'
+  },
+  devtool: 'source-map',
+  mode: 'development',
+  devServer: {
+    host: 'local-ip',
+    port: process.env.PORT || 9000,
+    hot: false,
+    liveReload: true,
+    static: {
+      directory: 'examples'
+    }
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env.INSPECTOR_VERSION': JSON.stringify(
+        process.env.INSPECTOR_VERSION
+      )
+    }),
+    new webpack.ProvidePlugin({
+      process: 'process/browser',
+      Buffer: ['buffer', 'Buffer']
+    })
+  ],
+  resolve: {
+    alias: {
+      three: 'super-three'
+    }
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        use: {
+          loader: 'babel-loader'
+          // options: {
+          //   presets: ['@babel/preset-env'],
+          // },
+        }
+      },
+      {
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader']
```

Comment:
style-loader will inject two style tags in the DOM corresponding to the two css files we import, similar to browserify-css